### ISSUE TITLE
Unknown Crash

### ISSUE THREAD

**Issue by PeterKW** [florisboard/florisboard#2045]
Can't quite figure out what caused the crash

#### Environment information
- FlorisBoard [0.4.0-alpha03](https://github.com/florisboard/florisboard/releases/v0.4.0-alpha03) (89)
- Device: Fairphone FP4 (FP4eea)
- Android: 11 (cn=R sdk=30)

#### Attached logs and stacktrace files
<details>
<summary>Detailed info (Debug log header)</summary>

```
======= SYSTEM INFO =======
Time                : 2022-09-02T18:16:10.283Z
Manufacturer        : Fairphone
Model               : FP4
Product             : FP4eea
Android             : 11 (cn=R sdk=30) [FP4.FP44.A.149.20220722]
ABIs                : [arm64-v8a, armeabi-v7a, armeabi]
Memory              : 5.56 GiB (75.83% used, 7.33 GiB max)
Font scale          : 1.3
Locales             : en-GB,en-US

======= APP INFO =======
Package             : dev.patrickgold.florisboard.beta
Name                : FlorisBoard Beta
Version             : 0.4.0-alpha03 (89)
Build type          : beta
Build commit hash   : 26650d2a00d509efbbd58bafa8606ba11bc5961f
Java heap memory    : 4.16 MiB (1.63% used, 256.00 MiB max)
Native heap memory  : 22.38 MiB (86.41% used, 25.90 MiB max)

======= FEATURE CONFIG =======
Smartbar enabled            : true
Suggestions enabled         : true
Inline autofill enabled     : true
Glide enabled               : true
Internal clipboard enabled  : true

```
</details>

<details>
<summary>1662142563886.stacktrace</summary>

```
java.lang.IllegalArgumentException: offset(-1) is out of bounds [0, 0]
	at androidx.compose.ui.text.MultiParagraph.requireIndexInRangeInclusiveEnd(MultiParagraph.kt:9)
	at androidx.compose.ui.text.TextLayoutResult.getWordBoundary--jx7JFs(TextLayoutResult.kt:2)
	at androidx.compose.foundation.text.selection.BaseTextPreparedSelection.getPrevWordOffset(TextPreparedSelection.kt:4)
	at androidx.compose.foundation.text.TextFieldKeyInput$process$2$5.invoke(TextFieldKeyInput.kt:5)
	at androidx.compose.foundation.text.selection.TextFieldPreparedSelection.deleteIfSelectedOr(TextPreparedSelection.kt:3)
	at androidx.compose.foundation.text.TextFieldKeyInput$process$2.invoke(TextFieldKeyInput.kt:95)
	at androidx.compose.foundation.text.TextFieldKeyInputKt$textFieldKeyInput$2$1.invoke(TextFieldKeyInput.kt:28)
	at androidx.compose.ui.input.key.KeyInputModifier.propagateKeyEvent-ZmokQxo(KeyInputModifier.kt:3)
	at androidx.compose.ui.platform.AndroidComposeView.dispatchKeyEvent(AndroidComposeView.android.kt:24)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
	at android.view.ViewGroup.dispatchKeyEvent(ViewGroup.java:1959)
	at com.android.internal.policy.DecorView.superDispatchKeyEvent(DecorView.java:476)
	at com.android.internal.policy.PhoneWindow.superDispatchKeyEvent(PhoneWindow.java:1863)
	at android.app.Activity.dispatchKeyEvent(Activity.java:4085)
	at androidx.core.app.ComponentActivity.superDispatchKeyEvent(Unknown Source:0)
	at androidx.core.app.ComponentActivity.dispatchKeyEvent(ComponentActivity.java:4)
	at com.android.internal.policy.DecorView.dispatchKeyEvent(DecorView.java:390)
	at android.view.ViewRootImpl$ViewPostImeInputStage.processKeyEvent(ViewRootImpl.java:5955)
	at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:5823)
	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:5318)
	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:5375)
	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:5341)
	at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:5493)
	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:5349)
	at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:5550)
	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:5322)
	at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:5375)
	at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:5341)
	at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:5349)
	at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:5322)
	at android.view.ViewRootImpl.deliverInputEvent(ViewRootImpl.java:8095)
	at android.view.ViewRootImpl.doProcessInputEvents(ViewRootImpl.java:8046)
	at android.view.ViewRootImpl.enqueueInputEvent(ViewRootImpl.java:8006)
	at android.view.ViewRootImpl$ViewRootHandler.handleMessage(ViewRootImpl.java:5083)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loop(Looper.java:223)
	at android.app.ActivityThread.main(ActivityThread.java:7715)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:952)

```
</details>

#### Short description
Describe what you were doing that could've led to the crash.

#### Steps to reproduce
1. Go to '…'
2. Click on '…'
3. Scroll down to '…'
4. See crash

**Comment by tsiflimagas** [comment]
Duplicate of #2026. It has been fixed in 0.4.0-alpha04. Closing thus.

**Comment by patrickgold** [comment]
Just to clarify: This issue is indeed a duplicate but not of #2026 but of #2001 and has not yet been addressed by me.
